validation/superchain-version_test.go

```diff
@@ -111,7 +112,7 @@ func checkSemverForContract(t *testing.T, contractName string, contractAddress *
 	actualSemver, err := getVersionWithRetries(context.Background(), common.Address(*contractAddress), client)
 	require.NoError(t, err, "Could not get version for %s", contractName)
 
-	require.Condition(t, func() bool { return isSemverAcceptable(desiredSemver, actualSemver) },
+	assert.Condition(t, func() bool { return isSemverAcceptable(desiredSemver, actualSemver) },
```

Comment:
Just curious to learn more about the require to assert change here?

Reply:
You got it, that's exactly my thinking here.